Dockerfile

```diff
@@ -15,7 +15,9 @@ RUN ./build.sh -v
 
 
 FROM debian:stretch
-RUN apt-get update && apt-get install -y mono-devel && apt-get clean all
+RUN apt-get update && apt-get install -y mono-devel curl && apt-get clean all
+RUN curl https://gist.githubusercontent.com/TheCakeIsNaOH/57254dc7f99b528d495e4f4e52de7f03/raw/ee936d6efddac485091ca6de833863f04ea77e39/Go%2520Daddy%2520Secure%2520Certificate%2520Authority%2520-%2520G2.crt >> /etc/ssl/certs/ca-certificates.crt
```

Comment:
Importing a certificate from some random gist isn't something I'd consider secure. Is there an official source for this certificate?

Reply:
Good point, I'm sure there is something more official

Reply:
Okay I found a official source here:
https://ssl-ccp.godaddy.com/repository/?origin=CALLISTO

It is the second certificate. I'll update my PR now.

Reply:
Done